apps/ejabberd/src/mod_muc_room.erl

```diff
@@ -4079,23 +4079,18 @@ invite_body_text(FromJID, Reason, Lang,
 
 
 -spec create_invite_message_elem(Inv :: jlib:xmlel(), Body :: jlib:xmlel(),
-        Passwd :: [jlib:xmlel()], RoomJID :: ejabberd:jid(), Reason :: binary()
+        Passwd :: [jlib:xmlel()], Reason :: binary()
         ) -> jlib:xmlel().
-create_invite_message_elem(InviteEl, BodyEl, PasswdEl, RoomJID, Reason)
+create_invite_message_elem(InviteEl, BodyEl, PasswdEl, Reason)
     when is_list(PasswdEl), is_binary(Reason) ->
-    BRoomJID = jlib:jid_to_binary(RoomJID),
     UserXEl = #xmlel{
         name = <<"x">>,
         attrs = [{<<"xmlns">>, ?NS_MUC_USER}],
         children = [InviteEl|PasswdEl]},
-    ConfXEl = #xmlel{
-        name = <<"x">>,
-        attrs = [{<<"xmlns">>, ?NS_XCONFERENCE}, {<<"jid">>, BRoomJID}],
```

Comment:
I was thinking about it but I found out that this namespace is used in XEP-0249: 'Direct MUC Invitations' which could be implemented in future. I can remove it anyway